python/tvm/contrib/pipeline_executor.py

```diff
@@ -533,11 +570,92 @@ def graph_executor_create(self, pipeline_mods, mod_config):
             The Modudle configuration.
         """
 
-        mods = []
-        for pipeline_mod in pipeline_mods:
-            mod = graph_executor.GraphModule(
-                pipeline_mod["default"](pipeline_mods[pipeline_mod]["dev"])
-            )
-            mods.append(mod.module)
+        # The module in pipeline_mods has a index information to identify the modules order
+        # in pipe line, the pipe line executor follow such asend order to run each module,
+        # but asend order of module is not guaranteed in pipeline_mod, here need to pre-allocate
+        # module list then put module in correct place follow the index value.
```

Comment:
This sentence is full of grammar errors, please fix all of them.

Even then, I don't understand what this sentence means. Please improve it.